### ISSUE TITLE
Email: Forest service image alt text is missing the visible text shown in the image.

### ISSUE THREAD

**Issue by JohnHewittCriterion** [nciinc/fs-permit-platform#613]
The images at the top of the email are linked images that have incorrect alt text. 
USDA should not say "logo" because they are not downloading the logo from the link. 
 
US Forest Service image is missing all the text displayed in the image. The alt text should say exactly what the visible image of text shows. 

![image.png](https://images.zenhubusercontent.com/5913513251aa2c30577de73e/f1ce750c-fa50-48e5-bcb1-4fb7d3bc0f39)

Linkage: CTF-96
WCAG: 1.1.1 Non-text Content

**Comment by sadlerw** [comment]
<img width="589" alt="screen shot 2018-03-22 at 11 25 10 am" src="https://user-images.githubusercontent.com/2058839/37783544-c1d52c48-2dc3-11e8-8a0c-128c9905b1fa.png">
